src/Microsoft.ApplicationInsights/Extensibility/W3C/W3CActivityExtensions.cs

```diff
@@ -1,385 +0,0 @@
-﻿namespace Microsoft.ApplicationInsights.Extensibility.W3C
```

Comment:
we cannot remove it - it's public. It could proxy calls to real Activity API when make sense and be noop in other cases